lib/matplotlib/tests/test_backend_pgf.py

```diff
@@ -338,3 +340,16 @@ def test_unknown_font(caplog):
         plt.savefig(BytesIO(), format="pgf")
     assert "Ignoring unknown font: this-font-does-not-exist" in [
         r.getMessage() for r in caplog.records]
+
+
+@check_figures_equal(extensions=["pdf"])
+@pytest.mark.parametrize("texsystem", ("pdflatex", "xelatex", "lualatex"))
+@pytest.mark.backend("pgf")
+def test_minus_signs_with_tex(fig_test, fig_ref, texsystem):
+    if not check_for(texsystem):
+        pytest.skip(texsystem + " is required")
+    mpl.rcParams["pgf.texsystem"] = texsystem
+    plt.figure(fig_test.number)
+    plt.figtext(.5, .5, "$-1$")
```

Comment:
Shouldn't there be an `r` before the string?
```suggestion
    plt.figtext(.5, .5, r"$-1$")
```

Reply:
Putting an "r" there would not change anything because there are not any escape sequences present. As soon as any LaTeX commands starting with a backslash are used, converting the string to a raw string with "r" would be necessary. But this is not the case here.